accounts/admin.py

```diff
@@ -32,18 +32,28 @@ class UserAdmin(admin.ModelAdmin):
     ordering = ("username",)
     fieldsets = (
         (None, {"fields": ("username", "pennid")}),
-        (("Personal info"), {"fields": ("first_name", "last_name", "email")}),
+        (("Personal info"), {"fields": ("preferred_name", "first_name", "last_name", "email")},),
         (
             ("Permissions"),
-            {"fields": ("is_active", "is_staff", "is_superuser", "groups", "user_permissions")},
+            {"fields": ("is_active", "is_staff", "is_superuser", "groups", "user_permissions",)},
         ),
         (("Important dates"), {"fields": ("last_login", "date_joined")}),
     )
 
 
+class PhoneNumberAdmin(admin.ModelAdmin):
```

Comment:
If we're not adding custom functionality to them we don't need to make the `PhoneNumberAdmin` and `EmailAdmin`